raster/r.what/main.c

```diff
@@ -255,8 +273,16 @@ int main(int argc, char *argv[])
     Cats = Vect_new_cats_struct();
     G_get_window(&window);
 
+    if (strcmp(opt.format->answer, "json") == 0) {
```

Comment:
Please consider creating an `enum` with members `JSON` and `PLAIN`, which can later be more easily checked in either `if...else` or (if appropriate) `switch` statement. See, e.g. #3534.

Reply:
Will update.

Reply:
Done.